components/dropzone/index.css

```diff
@@ -11,21 +11,191 @@ governing permissions and limitations under the License.
+  --spectrum-drop-zone-content-font-weight: var(--spectrum-bold-font-weight);
+  --spectrum-drop-zone-content-font-style: var(--spectrum-default-font-style);
+  --spectrum-drop-zone-content-font-size: var(--spectrum-font-size-300);
+  --spectrum-drop-zone-content-line-height: var(--spectrum-line-height-100);
+
+  --spectrum-drop-zone-content-background-color: var(--spectrum-accent-visual-color);
+  --spectrum-drop-zone-content-color: var(--spectrum-white);
+
+
+  /* cjk language support */
+  &:lang(ja),
+  &:lang(zh),
+  &:lang(ko) {
+    --spectrum-drop-zone-heading-font-size: var(--spectrum-drop-zone-cjk-title-size);
+  }
+
+}
+
+.spectrum-Dropzone {
+  position: relative;
+  box-sizing: border-box;
+  inline-size: var(--mod-drop-zone-width, var(--spectrum-drop-zone-width));
+  padding: calc(var(--mod-drop-zone-padding, var(--spectrum-drop-zone-padding)) - var(--mod-drop-zone-border-width, var(--spectrum-drop-zone-border-width)));
   text-align: center;
-  border-width: var(--spectrum-dropzone-border-width);
-  border-radius: var(--spectrum-dropzone-border-radius);
-  padding: var(--spectrum-dropzone-padding);
+  border-color: var(--spectrum-drop-zone-border-color, var(--spectrum-drop-zone-border-color));
+  border-width: var(--mod-drop-zone-border-width, var(--spectrum-drop-zone-border-width));
+  border-radius: var(--mod-drop-zone-corner-radius, var(--spectrum-drop-zone-corner-radius));
   border-style: dashed;
+  background-size: cover;
+
+  &::after {
+    content: "";
+    block-size: 100%;
+    inline-size: 100%;
+    position: absolute;
+    inset-inline-start: 0;
+    inset-block-start: 0;
+  }
+
+  .spectrum-IllustratedMessage {
+    pointer-events: none;
+    max-inline-size: var(--mod-drop-zone-content-maximum-width, var(--spectrum-drop-zone-content-maximum-width));
+  }
+
+  .spectrum-IllustratedMessage-illustration {
+    color: var(--highcontrast-drop-zone-illustration-color, var(--mod-drop-zone-illustration-color, var(--spectrum-drop-zone-illustration-color)));
+    margin-block-end: var(--mod-drop-zone-illustration-to-heading, var(--spectrum-drop-zone-illustration-to-heading));
+  }
+
+  .spectrum-IllustratedMessage-heading {
+    margin-block-end: var(--mod-drop-zone-heading-to-body, var(--spectrum-drop-zone-heading-to-body));
+    font-family: var(--mod-drop-zone-heading-font-family, var(--spectrum-drop-zone-heading-font-family));
+    font-weight: var(--mod-drop-zone-heading-font-weight, var(--spectrum-drop-zone-heading-font-weight));
+    font-style: var(--mod-drop-zone-heading-font-style, var(--spectrum-drop-zone-heading-font-style));
+    font-size: var(--mod-drop-zone-heading-font-size, var(--spectrum-drop-zone-heading-font-size));
+    line-height: var(--mod-drop-zone-heading-line-height, var(--spectrum-drop-zone-heading-line-height));
+    color: var(--mod-drop-zone-heading-color, var(--spectrum-drop-zone-heading-color));
+  }
+
+  .spectrum-IllustratedMessage-description {
+    pointer-events: initial;
```

Comment:
I don't think it is this line, but I noticed that there is no cursor on hover over the `Select a file` link when there was previously, is that something we want to keep?

Reply:
Well that line was supposed to fix that issue so I will take another look!